### PR TITLE
DOC correct gamma function in docstrings in scipy.stats

### PR DESCRIPTION
**Pull request by chrisb83**
docstrings in continuous_distributions.py:

Use `\Gamma` instead of `\gamma` to denote the Gamma functin `Special.gamma` as discussed in #8816. Only occurence of `\gamma` left is in `ncf_gen` where it denotes the log of `\Gamma`.

for `chi` and `chi2`, this overlaps with #8865.

### CODE REVIEW

File: scipy/stats/_continuous_distns.py [hunk 1]
```diff
@@ -905,9 +905,10 @@ class chi_gen(rv_continuous):
 
     .. math::
 
-        f(x, df) = \frac{x^{df-1} \exp(-x^2/2)}{2^{df/2-1} \gamma(df/2)}
+        f(x; df) = \frac{x^{df-1} \exp(-x^2/2)}{2^{df/2-1} \Gamma(df/2)}
```
Comment (pvanmulbregt) [inline scipy/stats/_continuous_distns.py]: Why the change `f(x, df)` -> `f(x; df)`? I.e. comma -> semi-colon?

Comment (chrisb83) [reply scipy/stats/_continuous_distns.py]: I think it ist better to seperate the shape parameters with a semicolon (Most common notation is probably to use subscripts like `f_\beta\) , see e.g. the incomplete beta function here https://en.m.wikipedia.org/wiki/Beta_function

However, one should probably agree on a consistent approach for all distributions here...

Comment (pvanmulbregt) [reply scipy/stats/_continuous_distns.py]: It wouldn't be an unreasonable distinction. However only two of the existing PDF functions (`norminvgauss` and `t`) use semi-colon, and the other 63 use comma. So probably better to keep this one as semi-colon.
[Even the referenced incomplete beta is inconsistent here.  The regularized incomplete beta function is written`$I_x(a, b)$` with the shape parameters inside the brackets, but the `x` is now a subscript!]

Comment (chrisb83) [reply scipy/stats/_continuous_distns.py]: agree, i changed the semicolon to comma again.

**Comment by chrisb83** [comment]
Not clear why python 2.7 build fails: 

`The log length has exceeded the limit of 4 MB (this usually means that the test suite is raising the same exception over and over).
The job has been terminated`

The following test is indeed repeated until the job is terminated:
`scipy/sparse/tests/test_base.py:4581: in _check_resiliency`

**Comment by pvanmulbregt** [comment]
I get the same error in a PR of mine(gh-8737). I think gh-8885 may have fixed it, disabling FutureWarning and PendingDeprecationWarning.  May require a rebase to pick up the change.

**Comment by chrisb83** [comment]
@pvanmulbregt : thanks for the hint, after the rebase, the build for 2.7 worked.

now there is an unrelated error for 3.6: 

> Error: Failed to download resource "gcc"

**Comment by rgommers** [comment]
> now there is an unrelated error for 3.6:

that looks like an intermittent CI issue, I've restarted the build

**Comment by chrisb83** [comment]
@rgommers hm, now it fails due to an import error.

**Comment by rgommers** [comment]
> @rgommers hm, now it fails due to an import error.

that seems to be due to the rebuild not being a full build, and us having bumped LAPACK to 3.4.0. Let's just ignore that one, it's unrelated to this PR.

**Comment by rgommers** [comment]
Okay merging. Thanks @chrisb83 and @pvanmulbregt
